tools/example-templates/additional-files/AuthorAttribute.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+[AttributeUsage(AttributeTargets.Class, AllowMultiple = true)]
+public class AuthorAttribute : Attribute
```

Comment:
Can we use autoprops now for attributes? If so, it would be good to do so.

```csharp
public string Name { get; }
public AuthorAttribute(string name) => Name= name;
```

Reply:
Agreed; changed in both the addition-file and in attributes.md where this type is declared.